content/en/blog/2023/announcing-notation-v1.0.0.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Notation v1.0.0 is available!
+author:  "Notation Release Team"
+date: 2023-07-15
+draft: false
+---
+
+The Notary Project maintainers are thrilled to announce the first stable release of Notation, including [notation v1.0.0](https://github.com/notaryproject/notation/releases/tag/v1.0.0), [notation-go v1.0.0](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0), [notation-core-go v1.0.0](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0) which are ready for production use! 
+
+
+## What is Notation
+
+As containers and cloud native artifacts become the common unit of deployment, users want to know the artifacts in their environments are authentic and unmodified. The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains through signing and verification, signature portability, and key management. It aims to solve the core issue of trusting content within, and across registries. 
+
+Notation (previously known as Notary v2) is a sub-project of Notary Project. It has notation CLI and two Golang libraries, which are implemented from the Notary Project specifications. Notation was started in [Dec 2019](https://github.com/notaryproject/meeting-notes/blob/main/meeting-notes-2019.md#notary-v2-kickoff-meeting) and the code case has matured through a series of minor and RC releases over the last four years, the first release [v0.7.0-alpha.1](https://notaryproject.dev/blog/2021/announcing-notation-alpha1/) was available in Oct 2021 and reached [v1.0.0-RC.7](https://notaryproject.dev/blog/2023/announcing-notation-rc6/) in May 2023.
+
+## Highlights
+
+There are some remarkable new features and updates in Notation v1.0.0. This blog post walks through the major updates in this release.
+
+### Improved signing and verification functionalities
+
+From the software producer's perspective, signing an artifact is the way to ensure authenticity and trust when distributing a software to consumers. Notation v1.0.0 added new capabilities to signing experience. 
```

Comment:
```suggestion
From the software producer's perspective, signing an artifact is the way to ensure authenticity and trust when distributing a software to consumers. Notation v1.0.0 provides the follow capabilities for the signing experience. 
```
I would refrain from using the word "Added" as it implies we added something in v1.0.0 as compared to RC-7, which we didn't.